src/team_comm_tools/feature_builder.py

```diff
@@ -501,6 +522,7 @@ def featurize(self) -> None:
             print("All Done!")
             
             # Store column names of what we generated, so that the user can easily access them
+            # TODO --- this needs to be updated if the user brings their own LIWC, because the custom LIWC features are not in `self.chat_features`.
```

Comment:
@sundy1994 Can you please update the `chat_features` property of the FeatureBuilder object whenever the user generates custom LIWC features? Otherwise, those new column names will not show up when the user tries to check the names of the chat features.

Reply:
DONE